### ISSUE TITLE
Issues with "Managing your Account" in docs.anaconda.org

### ISSUE THREAD

**Issue by gpfreitas** [Anaconda-Platform/docs.anaconda.org#265]
1. __Excessive detail.__ On [Create free account](http://docs.anaconda.org/managing-account.html#CreateFreeAccount) (by the way, shouldn't it be "Create _a_ free account"?) , after we tell the user which form (the Sign Up tab) to fill in, I think we should just say "Follow the on-screen instructions". It should be enough and it's robust to possible changes in the form.

2. __Wrong screenshot.__ The [toolbar screenshot](http://docs.anaconda.org/media/img/Anaconda-Cloud-User-Toolbar.jpg) is outdated. The new toolbar looks like this:

![image](https://cloud.githubusercontent.com/assets/1276576/14656811/46f5dff6-064f-11e6-8bd4-838501796f64.png)

There is also a bullet list describing the toolbar in the docs. I don't think we should do that sort of thing. Shouldn't we just provide a description of the button when the mouse cursor hovers over that? It doesn't help mobile users though. Hmmm. I'm just thinking that this sort of error is going to crop up more and more often, as the objects we are documenting change: the more detail we provide in the docs, the more we will have to update them. The more we can rely on some sort of "self-documentation" of specific elements of a product, the higher is the level at which we can write docs, and the more robust the docs will be. Thoughts, @electronwill , @kerrywatson1 ?

3. __Wrong instructions__. The  [Upgrade or downgrade plan](http://docs.anaconda.org/managing-account.html#UpgradeOrDowngradePlan) instructions are wrong because the toolbar has changed. Same thing with the [Create organization](http://docs.anaconda.org/managing-account.html#CreateOrganization) instructions.

...

@kerrywatson1 this is will be very slow if I first describe everything that I think should be changed... Should I just submit a pull request with a bunch of changes and we discuss the changes there? Then I don't have to describe the changes, I can just _make_ the changes in my branch, and you can just look at the diff, hopefully infer the reason for the change, verify it if necessary, make your edits and wrap it up. How does that sound?

**Comment by kerrywatson1** [comment]
Yes, make changes and we'll review the diff.

On Tuesday, April 19, 2016, Guilherme Freitas <notifications@github.com>
wrote:

>
>    1.
>
>    *Excessive detail.* On Create free account
>    <http://docs.anaconda.org/managing-account.html#CreateFreeAccount> (by
>    the way, shouldn't it be "Create *a* free account"?) , after we tell
>    the user which form (the Sign Up tab) to fill in, I think we should just
>    say "Follow the on-screen instructions". It should be enough and it's
>    robust to possible changes in the form.
>    2.
>
>    *Wrong screenshot.* The toolbar screenshot
>    <http://docs.anaconda.org/media/img/Anaconda-Cloud-User-Toolbar.jpg>
>    is outdated. The new toolbar looks like this:
>
> [image: image]
> <https://cloud.githubusercontent.com/assets/1276576/14656811/46f5dff6-064f-11e6-8bd4-838501796f64.png>
>
> There is also a bullet list describing the toolbar in the docs. I don't
> think we should do that sort of thing. Shouldn't we just provide a
> description of the button when the mouse cursor hovers over that? It
> doesn't help mobile users though. Hmmm. I'm just thinking that this sort of
> error is going to crop up more and more often, as the objects we are
> documenting change: the more detail we provide in the docs, the more we
> will have to update them. The more we can rely on some sort of
> "self-documentation" of specific elements of a product, the higher is the
> level at which we can write docs, and the more robust the docs will be.
> Thoughts, @electronwill <https://github.com/electronwill> , @kerrywatson1
> <https://github.com/kerrywatson1> ?
>
>    1. *Wrong instructions*. The Upgrade or downgrade plan
>    <http://docs.anaconda.org/managing-account.html#UpgradeOrDowngradePlan>
>    instructions are wrong because the toolbar has changed. Same thing with the Create
>    organization
>    <http://docs.anaconda.org/managing-account.html#CreateOrganization>
>    instructions.
>
> ...
>
> @kerrywatson1 <https://github.com/kerrywatson1> this is will be very slow
> if I first describe everything that I think should be changed... Should I
> just submit a pull request with a bunch of changes and we discuss the
> changes there? Then I don't have to describe the changes, I can just
> *make* the changes in my branch, and you can just look at the diff,
> hopefully infer the reason for the change, verify it if necessary, make
> your edits and wrap it up. How does that sound?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/Anaconda-Platform/docs.anaconda.org/issues/265>
>


-- 
_______________________________

Kerry Watson
Documentation manager

[image: Continuum Analytics] <http://continuum.io>

**Comment by electronwill** [comment]
Yes, PRs for improvements are very welcome! Thank you for offering help with these docs.

You also make a great point about high level robust docs instead of detailed ones that have to change often! As you said, we do still have to make sure they're correct and they're specific enough to work.

It is best to make the PRs at the current docs location: https://github.com/Anaconda-Platform/anaconda-server/tree/develop/docs/content

This repo is deprecated, as explained in the readme at https://github.com/Anaconda-Platform/docs.anaconda.org .

**Comment by gpfreitas** [comment]
@electronwill , is it possible to make those "Click here to edit the docs" links to point to the current docs location (within anaconda-server docs)? I used the old one because all I did was click on that button... which is a _very_ nice workflow, by the way!

**Comment by electronwill** [comment]
Oh! I didn't realize those "Click here to edit the docs" links were there and pointing to this repo. Yes, let me see about fixing that...

**Comment by electronwill** [comment]
Ah, I understand now. The docs are now in the private repo Anaconda-Platform/anaconda-server , so if we point the links there it will just be a 404 for everyone who doesn't have access to the repo. We hope to migrate docs.anaconda.org into docs.continuum.io during Q2 2016. For now, I've updated the readme in this repo to explain the situation more clearly, and welcome people who don't have access to the private repo to suggest docs changes in the public "support" repo: https://github.com/Anaconda-Platform/docs.anaconda.org/pull/266
